lightning/src/util/events.rs

```diff
@@ -403,6 +403,31 @@ pub enum Event {
 		/// May contain a closed channel if the HTLC sent along the path was fulfilled on chain.
 		path: Vec<RouteHop>,
 	},
+	/// Indicates a request to open a new channel by a peer. The
+	/// [`ChannelManager::respond_to_open_channel_request`] function should be called to respond to the request.
+	///
+	/// The event is only triggered when a new open channel request is received and the
+	/// [`UserConfig::manually_accept_inbound_channels`] config flag is set to true.
+	///
+	/// [`ChannelManager::respond_to_open_channel_request`]: crate::ln::channelmanager::ChannelManager::respond_to_open_channel_request
+	/// [`UserConfig::manually_accept_inbound_channels`]: crate::util::config::UserConfig::manually_accept_inbound_channels
+	OpenChannelRequest {
+		/// The temporary channel ID of the channel requested to be opened.
+		///
+		/// When accepting or rejecting to the request, the `temporary_channel_id` should be passed back to the ChannelManager
+		/// with [`ChannelManager::respond_to_open_channel_request`].
+		///
+		/// [`ChannelManager::respond_to_open_channel_request`]: crate::ln::channelmanager::ChannelManager::respond_to_open_channel_request
+		temporary_channel_id: [u8; 32],
+		/// The node_id of the counterparty requesting to open the channel.
+		counterparty_node_id: PublicKey,
+		/// The genesis hash of the blockchain where the channel is requested to be opened.
+		chain_hash: BlockHash,
+		/// The channel value of the requested channel.
+		funding_satoshis: u64,
+		/// The amount to push to the counterparty if the channel request is accepted, in milli-satoshi.
```

Comment:
That's us! Maybe say something like "the amount the counterparty will give us if the channel is opened" or talk about it as "our starting balance"

Reply:
Oh my bad, should have known that!